clang/test/SemaCUDA/spirv-attrs-diag.cu

```diff
@@ -0,0 +1,34 @@
+// RUN: %clang_cc1 -triple spirv64 -aux-triple x86_64-unknown-linux-gnu \
+// RUN:   -fcuda-is-device -verify -fsyntax-only %s
+
+#include "Inputs/cuda.h"
```

Comment:
Probably should just locally define __global__ instead of depending on the full input header